### PR TITLE
Add binding init

### PR DESCRIPTION
**Pull request by wojtek717**

### CODE REVIEW

File: Sources/BottomSheet/BottomSheet.swift [hunk 1]
```diff
@@ -50,11 +53,23 @@ public extension View {
...
     }
-
+    
+    /// Presents a sheet.
+    /// - Parameter sheetContent: A closure that returns the content of the sheet.
+    func bottomSheet<SheetContent: View>(
+        @ViewBuilder sheetContent: () -> SheetContent
+    ) -> BottomSheet<Self, SheetContent> {
+        BottomSheet(
+            isPresented: .constant(true),
```
Comment (samdogg7) [inline Sources/BottomSheet/BottomSheet.swift:63]:
```suggestion
    func bottomSheet<SheetContent: View>(
        isPresented: Binding<Bool> = .constant(true),
        @ViewBuilder sheetContent: () -> SheetContent
    ) -> BottomSheet<Self, SheetContent> {
        BottomSheet(
            isPresented: isPresented,
```

File: Sources/BottomSheet/BottomSheet.swift [hunk 2]
```diff
@@ -50,11 +53,23 @@ public extension View {
...
+            content: { self },
+            sheetContent: sheetContent
+        )
+    }
 }
 
 private struct ExampleView: View {
     let colors: [Color] = [.red, .orange, .yellow, .green, .blue, .indigo, .purple]
+    @State private var isShowingSheet = false
```
Comment (samdogg7) [inline Sources/BottomSheet/BottomSheet.swift:72]: After implementing previous suggestion, this can then be passed into the `bottomSheet`

**Review by samdogg7 — CHANGES_REQUESTED** [review]
Looks good! One change request

File: Sources/BottomSheet/BottomSheet.swift (hunk 2 above)
Comment (wojtek717) [reply Sources/BottomSheet/BottomSheet.swift]: Removed this @State from this example.

File: Sources/BottomSheet/BottomSheet.swift (hunk 1 above)
Comment (wojtek717) [reply Sources/BottomSheet/BottomSheet.swift]: Moved to one init.

**Review by sam-doggett-simplisafe — APPROVED** [review]
Good stuff 👍 I actually overlooked that there were two `func bottomSheet<SheetContent: View>` initially

**Review by samdogg7 — APPROVED** [review]

**Pull request merged**